runtime/manifest-parser.peg

```diff
@@ -156,6 +176,16 @@ RecipeView
     }
   }
 
+RecipeSlot
+  = "renders" ref:(whiteSpace ViewRef)? name:(whiteSpace Name)? eolWhiteSpace
```

Comment:
According to what you suggested, it actually is a view reference:

recipe
 renders [view] as localSlotName
 SomeParticle
....


so for example in the strategy explorer i get for example:

recipe
  ...
  map 'test-plan-arc:2' as view2
  ...
  renders 'test-plan-arc:2' as slot2
  ...